windows/build_release.bat

```diff
@@ -14,6 +14,9 @@ set VCDIR=%VCToolsInstallDir%
 set SDKDIR=.
 set MSVC_AR=%VCToolsInstallDir%\bin\Hostx64\x64\lib.exe
 
+rem enable autodetection in LDC, so it doesn't mix x86/x64 libs
+set VSINSTALLDIR=
```

Comment:
This makes LDC use LLD and the MinGW libs. Use this for proper x86/x64 setup:
```batch
set LDC_VSDIR=%VSINSTALLDIR%
set VSINSTALLDIR=
```

Reply:
> This makes LDC use LLD and the MinGW libs.

Oops, indeed. Thanks for the tip.